### PR TITLE
Prepare examples of prettier formatted code

### PR DESCRIPTION
**Pull request by JuliaGerasymenko**

### CODE REVIEW

File: .eslintrc.yml [hunk 1]
```diff
@@ -1 +1,10 @@
-extends: 'metarhia'
+extends:
+  - metarhia
+  - plugin:prettier/recommended
+  - prettier/react
```
Comment (aqrln) [inline .eslintrc.yml]: You don't need any of these except `metarhia` and `plugin:prettier/recommended`.

File: test/id.js [hunk 2]
```diff
@@ -15,79 +16,107 @@ const config = {
...
-  ],
-  'common.crcSID': [
-    [
-      config,
-      common.generateKey(
-        config.length - 4,
-        config.characters
-      ),
-      result => result.length === 4,
+metatests.case(
```
[the remark this answers is not in the document]
Comment (nechaido) [reply test/id.js]: I am perfectly fine with proposed changes.

File: .prettierignore [hunk 3]
```diff
@@ -0,0 +1,3 @@
+node_modules/
```
Comment (belochub) [inline .prettierignore]: There is no need to add `node_modules/` to this file; it's ignored by default.

Comment (belochub) [reply .prettierignore]: Also, you still haven't addressed the https://github.com/metarhia/common/pull/243#discussion_r247047923; you must specify the files that should not be formatted by prettier in this file.

File: README.md [hunk 4]
```diff
@@ -15,64 +15,84 @@ $ npm install @metarhia/common
 ```
```
Comment (belochub) [inline README.md]: `README.md` should also be ignored since it is autogenerated.

File: package.json [hunk 5]
```diff
@@ -29,7 +29,8 @@
   ],
   "scripts": {
     "test": "eslint . && metatests test/",
-    "lint": "eslint .",
+    "lint": "eslint . && prettier -c \"**/*.js\" \"**/*.json\" \"**/*.md\" \".*rc\"",
```
Comment (belochub) [inline package.json]: You should also add `*/**.yml` files here and below.

**Comment by belochub** [comment]
Landed in https://github.com/metarhia/common/commit/d018c64c21849de42bb90b9cb243e5b1fb176e0e.

**Pull request closed without merging**